sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs

```diff
@@ -221,5 +233,18 @@ public void assertHappyPath(SmsSendResult sendResult)
             Assert.AreEqual(202, sendResult.HttpStatusCode);
             Assert.IsFalse(string.IsNullOrWhiteSpace(sendResult.MessageId));
         }
+
+        public void assertRawResponseHappyPath(Stream contentStream)
+        {
+            if (contentStream.Length > 0)
```

Comment:
nit: you can invert the condition to reduce the long nested code and avoid usage of empty `return`